string robotName
float32 dist